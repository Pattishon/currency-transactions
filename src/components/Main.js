import React from "react";
import AddTransaction from "./AddTransaction";
import Converter from "./Converter";
import TransactionsList from "./TransactionsList";
import TransactionsSum from "./TransactionsSum";
import TransactionsMax from "./TransactionsMax";

const Main = () => {
  return (
    <div className="app">
      <h1 className="title">Transakcje walutowe</h1>
      <div className="transactions">
        <div>
          <Converter />
          <AddTransaction />
          <TransactionsList />
          <TransactionsSum />
        </div>
        <TransactionsMax />
      </div>
    </div>
  );
};

export default Main;
