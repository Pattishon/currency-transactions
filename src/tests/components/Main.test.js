import React from "react";
import { shallow } from "enzyme";
import Main from "../../components/Main";

test("should render Main correctly", () => {
  const wrapper = shallow(<Main />);
  expect(wrapper).toMatchSnapshot();
});
