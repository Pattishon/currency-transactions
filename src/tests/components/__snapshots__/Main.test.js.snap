// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Main correctly 1`] = `
<div
  className="app"
>
  <h1
    className="title"
  >
    Transakcje walutowe
  </h1>
  <div
    className="transactions"
  >
    <div>
      <Connect(Converter) />
      <Connect(AddTransaction) />
      <Connect(TransactionsList) />
      <Connect(TransactionsSum) />
    </div>
    <Connect(TransactionsMax) />
  </div>
</div>
`;
